3.Tests/UI/Features/RecycleBin/DeleteAll.feature

```diff
@@ -3,7 +3,7 @@ Feature: Empty recycle bin
     Background:
         Given the user is logged in
 
-    @Regression
+    @Regression @prueba
```

Comment:
What the @prueba tag is for? Should be removed right?

Reply:
Yeah, it was to do isolated test on this one, but now is deleted. Thank you.